pkg/edged/edged.go

```diff
@@ -57,6 +57,7 @@ import (
 	utilpod "github.com/kubeedge/kubeedge/pkg/edged/util/pod"
 	"github.com/kubeedge/kubeedge/pkg/edged/util/record"
 	"github.com/kubeedge/kubeedge/pkg/metamanager"
+	"sync"
```

Comment:
Please move the "sync" around the 

"fmt"
"time"

Reply:
done.